docs/how-to-run-bench-gimlet.adoc

```diff
@@ -0,0 +1,299 @@
+an actual gimlet, but with nothing other than power connected to the backplane.
+
+In examples used here, we had two K.2 cards each with a network connection to a
+different network.  This is not a requirement for running a bench gimlet.  Some
+examples will be different depending on if you have one or two network interfaces.
+
+If you're okay with the omicron ip pools (both internal services & instances)
+existing on the same L2 segment with the network used to access the gimlet, then
+just one NIC is fine.  That's basically the setup described in the main how-to-run doc.
+
+In our examples, one network will be used for connecting to the host OS running
+on the gimlet, and the second network will be used for connecting to Nexus as well
+as allow instances running on Omicron to connect to the outside.
+
+Whatever network you plan on using for the gimlet, you will need a contiguous
+range of IP addresses.  This range will be split into two segments. One segment
+will be for internal services, the other for external.  The required minimum
+number of IPs for each segment is decided by requirements in Omicron, and the
+values here may be out of date.
+
+== Overview
+
+For setting up a build system, follow the instructions in
+xref:how-to-run.adoc[How to Run].
+
+You will need to compile Omicron somewhere else, then copy the files over
+to the gimlet.
+
+== Gimlet setup
+
+You gimlet should have the M.2 devices formatted and setup with 4k sector
+sizes.  Info on how to do that is:  <TBD>
+
+== A helios image for your bench gimlet
+
+To run Omicron, you require a helios image on your gimlet constructed with
+some required libraries.  If your gimlet is lacking these, you will need to
+make and install a special build on your gimlet.
+To do that:
+Clone the helios repo: https://github.com/oxidecomputer/helios.git
+Check the helios repo README, as things may have changed.
+
+First, run gmake setup:
+[source,text]
+----
+cd helios
+gmake setup
+----
+
+Once that finishes, you are ready to build your helios image.  Below is an
+example command used to build the image for gimlet-sn21.
+
+The optional `-P` argument arranges for anything in that directory to be laid
+on top of the base image, which we presently use to, say, add in the
+sled-agent SMF manifest and binaries etc.  For example if you `mkdir /tmp/proto`
+and create `/tmp/proto/usr/bin/blah` then passing the `-P /tmp/proto` to your
+build command would make `/tmp/proto/usr/bin/blah` appear in the image at
+`/usr/bin/blah`
+
+You should update `-N` (name of your image) and `-o` (output location of build
+files) for your setup and needs.
+
+[source,text]
+----
+ds=$(TZ=US/Pacific date +%Y%m%d-%H%M%S)
+./helios-build experiment-image \
+  -N "gimlet-sn21-$ds" \
+  -p helios-netdev=https://pkg.oxide.computer/helios-netdev \
+  -o "/net/catacomb/data/staff/dock/gimlet-sn21/os-$ds" \
+  -F optever=0.23 \
+  -B \
+  -P alanproto
+----
+
+The `-F optever=0.23` is subject to change.
```

Comment:
Perhaps suggest checking `tools/opte_version` in this repo for the latest version to use?